azurelinuxagent/ga/monitor.py

```diff
@@ -130,6 +129,7 @@ def __init__(self):
         self.last_host_plugin_heartbeat = None
         self.last_imds_heartbeat = None
         self.protocol = None
+        self.protocol_util = None
```

Comment:
Something for future - Should this be private? We dont need to expose this outside. (applies to other attributes as well)